test/Basic/PendulumWithDamper.jl

```diff
@@ -41,7 +41,7 @@ PendulumWithDamp = Model(
 
 #@showModel PendulumWithDamp
 
-pendulumWithDamper = @instantiateModel(PendulumWithDamp, unitless=true, log=false, logStateSelection=false, logCode=true)
+pendulumWithDamper = @instantiateModel(PendulumWithDamp, unitless=true, log=false, logStateSelection=false, logCode=false, FloatType = Float32)
```

Comment:
No update of final states necessary?

Reply:
No, surprisingly not. The main values (r_rel, R_rel, etc.) of an Object3D are still `Float64`. This needs to be changed in an upcoming PR.